cookbook/form/data_transformers.rst

```diff
@@ -161,17 +161,17 @@ to and from the issue number and the ``Issue`` object::
     namespace AppBundle\Form\DataTransformer;
 
     use AppBundle\Entity\Issue;
-    use Doctrine\Common\Persistence\EntityManager;
+    use Doctrine\Common\Persistence\ObjectManager;
```

Comment:
the changes in this document seems not to be related to this PR (but to #5666 instead, which I just merged).